src/Pages/NotificationsSettings/index.tsx

```diff
@@ -0,0 +1,140 @@
+import { ToggleSwitchList } from '@/Components/ToggleSwitchList';
+import styled from 'styled-components';
+import { BoldDivider } from '@/Components/Common/Divider/BoldDivider';
+import { useNotificationsSetting } from '@/Hooks/notifications/useNotificationsSetting';
+import { Loading } from '@/Assets';
+import { KeywordsSettingForm } from './KeywordsSettingForm';
+
+export const NotificationsSettings = () => {
+  const { data: notificationsSetting, isLoading } = useNotificationsSetting();
+
+  return (
+    <NotificationsSettingsLayout>
+      {isLoading ? (
+        <Loading />
+      ) : (
+        <>
+          <AllOnOffSettingSection>
+            <ToggleSwitchList
+              label="전체 알림"
+              description="제목"
+              defaultChecked={notificationsSetting?.allConfig.on}
+              type="ALL_CONFIG"
+            />
+          </AllOnOffSettingSection>
+          <BoldDivider $rowHeight={2} />
+          <RecruitmentSettingsSection>
+            <SettingTitleBox>모집 공고</SettingTitleBox>
+            <ToggleSwitchList
+              label="모집 공고 알림"
+              description="권한 허용 후, 선호하는 항목을 고르면, 해당 모집 공고가 업로드 됐을 시 알림이 갑니다."
+              defaultChecked={notificationsSetting?.recruitmentConfig.on}
+              type="RECRUITMENT_CONFIG"
+              disabled={!notificationsSetting?.allConfig.on}
+            />
+            <KeywordsSettingForm
+              values={{
+                stackIds: notificationsSetting?.recruitmentConfig.stackKeywords.map(
+                  (stackKeyword: { stackId: number; name: string }) => stackKeyword.stackId,
+                ),
+                positionIds: notificationsSetting?.recruitmentConfig.positionKeywords.map(
+                  (positionKeyword: { positionId: number; name: string }) => positionKeyword.positionId,
+                ),
+                categoryIds: notificationsSetting?.recruitmentConfig.categoryKeywords.map(
+                  (categoryKeyword: { categoryId: number; name: string }) => categoryKeyword.categoryId,
```

Comment:
```suggestion
                stackIds: notificationsSetting?.recruitmentConfig?.stackKeywords?.map(
                  (stackKeyword: { stackId: number; name: string }) => stackKeyword.stackId,
                ),
                positionIds: notificationsSetting?.recruitmentConfig?.positionKeywords?.map(
                  (positionKeyword: { positionId: number; name: string }) => positionKeyword.positionId,
                ),
                categoryIds: notificationsSetting?.recruitmentConfig?.categoryKeywords?.map(
                  (categoryKeyword: { categoryId: number; name: string }) => categoryKeyword.categoryId,
```
`stackKeywords`, `positionKeywords`, `categoryKeywords`가 API 구조상 undefined가 내려올 일은 없겠지만
중첩객체에서 키가 없거나 혹시 모른 undefined 값에 대비해서 `?.` 옵셔널 체이닝을 사용하는게 좋을 것 같습니다!

Reply:
반영하였습니다👍